core/definitions.mk

```diff
@@ -7,7 +7,7 @@ define get-allowed-api-levels-regex
 endef
 
 define find-apk-for-pkg
-$(shell $(GAPPS_BUILD_SYSTEM_PATH)/find_apk.sh "$(GAPPS_SOURCES_PATH)" $(PLATFORM_SDK_VERSION) $(PRODUCT_AAPT_PREF_CONFIG) $(2) $(1) $(GAPPS_FORCE_MATCHING_DPI) $(GAPPS_AAPT_PATH))
+$(shell $(GAPPS_BUILD_SYSTEM_PATH)/find_apk.sh "$(GAPPS_SOURCES_PATH)" $(PLATFORM_SDK_VERSION) "$(PRODUCT_AAPT_PREF_CONFIG)" $(2) $(1) $(GAPPS_FORCE_MATCHING_DPI) $(GAPPS_AAPT_PATH))
```

Comment:
True, I know. But it is a good habit to quote (maintaining all the regular Open GApps buildscripts that use posix shell, I know how much trouble can be avoided by using them consistently ;-) )